### ISSUE TITLE
Unsupported Operation (MEAN) while trying to apply GpuDelegate to tflite 

### ISSUE THREAD

**Issue by Mr-Grieves** [tensorflow/tensorflow#28673]
<em>Please make sure that this is a bug. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:bug_template</em>

**System information**
- I am writing a custom Android app, built in Android Studio using the method mentioned [here](https://www.tensorflow.org/lite/performance/gpu#android_with_android_studio)
- Host system: Linux Ubuntu 16.04
- Target Device: Android s8+
- tflite installed from (source or binary): nightly AAR (org.tensorflow:tensorflow-lite:0.0.0-nightly)
- tflite-gpu installed from (source or binary): nightly AAR (org.tensorflow:tensorflow-lite-gpu:0.0.0-nightly)
- GPU model and memory: Adreno 540

**Describe the current behavior**
When I attempt to load my .tflite model on the Android (after applying the GpuDelegate as described [here](https://www.tensorflow.org/lite/performance/gpu#android)), it fails saying that 'MEAN' is an unsupported operation (see traceback below)

**Describe the expected behavior**
The net loads and runs fine if I do not attempt to apply the GPU delegate before the attempting to load it. The only difference in the code is the following lines:
```
c.delegate = new GpuDelegate();
c.tfliteOptions.addDelegate(c.delegate);
```

**Code to reproduce the issue**
Unfortunately I cannot provide my .tflite file, but it obviously contains a 'MEAN' operation. Is there a script anywhere in the tflite repo that prints out all the operations contained in a .tflite file? I looked briefly but was unable to find one.

**Other info / logs**
```
05-13 12:09:56.101 3348-3348/org.qus.viewqualTFLite E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.qus.viewqualTFLite, PID: 3348
    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.qus.viewqualTFLite/org.qus.viewqualTFLite.ViewQualActivity}: java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: Next operations are not supported by GPU delegate:
    MEAN: Operation is not supported.
    First 43 operations will run on the GPU, and the remaining 7 on the CPU.TfLiteGpuDelegate Prepare: Tensor ref has unsupported number of dimensions: 5Node number 50 (TfLiteGpuDelegate) failed to prepare.
    
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2955)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3030)
        at android.app.ActivityThread.-wrap11(Unknown Source:0)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1696)
        at android.os.Handler.dispatchMessage(Handler.java:105)
        at android.os.Looper.loop(Looper.java:164)
        at android.app.ActivityThread.main(ActivityThread.java:6938)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:327)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1374)
     Caused by: java.lang.IllegalArgumentException: Internal error: Failed to apply delegate: Next operations are not supported by GPU delegate:
    MEAN: Operation is not supported.
    First 43 operations will run on the GPU, and the remaining 7 on the CPU.TfLiteGpuDelegate Prepare: Tensor ref has unsupported number of dimensions: 5Node number 50 (TfLiteGpuDelegate) failed to prepare.
    
        at org.tensorflow.lite.NativeInterpreterWrapper.applyDelegate(Native Method)
        at org.tensorflow.lite.NativeInterpreterWrapper.init(NativeInterpreterWrapper.java:83)
        at org.tensorflow.lite.NativeInterpreterWrapper.<init>(NativeInterpreterWrapper.java:60)
        at org.tensorflow.lite.Interpreter.<init>(Interpreter.java:224)
        at org.qus.viewqualTFLite.TensorFlowQUSRunner.create(TensorFlowQUSRunner.java:154)
        at org.qus.viewqualTFLite.ViewQualActivity.onCreate(ViewQualActivity.java:200)
        at android.app.Activity.performCreate(Activity.java:7183)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1220)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2908)
            ... 9 more
```

**Comment by Mr-Grieves** [comment]
I'm guessing that I should open a 2nd issue for this, but I am experiencing that exact same failure while trying to run a different net, this time with the 'RESIZE_NEAREST_NEIGHBOR' as the unsupported op. Traceback:

**Comment by talumbau** [comment]
You can produce a listing of all of the ops and tensors in your .tflite file as an HTML file using the visualize tool:

bazel run //tensorflow/lite/tools:visualize model.tflite visualized_model.html

That should help you see where the MEAN op is.

**Comment by impjdi** [comment]
@Mr-Grieves 

If MEAN is used as part of batch norm, it needs to be removed (at inference, batch size will be most probably 1).  If MEAN is used for something else, you probably can't remove it, and we have internal requests to accommodate those "accumulators", but it may take some time for us to write it in shader code.  I will update the thread if we release it.

**Comment by impjdi** [comment]
this is quite an old bug ... 3 years old... and i'm not sure why this is being revived.  gonna close this.
